libraries/rating-bar/build.gradle

```diff
@@ -1,16 +1,18 @@
 apply plugin: 'com.android.library'
 apply plugin: 'kotlin-android'
 apply plugin: 'kotlin-android-extensions'
+apply plugin: 'kotlin-kapt'
```

Comment:
library module should be Databinding and kapt free

Reply:
We will discuss this later.